sql/service/src/main/java/org/apache/spark/sql/service/cli/operation/OperationLog.java

```diff
@@ -154,24 +168,26 @@ synchronized void write(String msg) {
       return readResults(maxRows);
     }
 
-    void remove() {
+    synchronized void remove() {
```

Comment:
Q: Was it a bug before to not have it synchronized?

Reply:
> Q: Was it a bug before to not have it synchronized?

I make some mistake when I copy `OperationLog`,
First I copy it from hive-1.2.1, then it have `org.apache.commonn.log`.
Now I get it from hive-2.3, Not sure why add this, but works well.